webapp.py

```
@@ -283,5 +292,46 @@ def set_runes_preference():
+
+
+
+@api.route("/actions/createlobby", methods=["POST"])
+@ensure_connection
+def create_lobby():
+    lobbytype = flask.request.json['lobbytype']
+    try:
+        lobby.create_lobby(state.connection, lobbytype)
+        return empty_success_response()
+
+    except Exception as e:
```

Ruff (0.14.0)

Do not catch blind exception: `Exception`

(BLE001)